content/sips/sip-337.md

```diff
@@ -33,58 +33,58 @@ Despite achieving moderate levels of success since launching with ~$150M of open
 
-*******************************************Cross-margin and multi-collateral accounts******************************************* 
+**\*\*\*\***\*\*\*\***\*\*\*\***\*\*\***\*\*\*\***\*\*\*\***\*\*\*\***Cross-margin and multi-collateral accounts**\*\*\*\***\*\*\*\***\*\*\*\***\*\*\***\*\*\*\***\*\*\*\***\*\*\*\***
 
-At a high level, cross-margin and multi-collateral support are simplified via user accounts represented by an NFT. To ensure homogenous distribution of risk to LPs throughout the cross margin system, individual markets are aggregated into a cross-marginable *supermarket*. 
+At a high level, cross-margin and multi-collateral support are simplified via user accounts represented by an NFT. To ensure homogenous distribution of risk to LPs throughout the cross margin system, individual markets are aggregated into a cross-marginable _supermarket_.
 
-*Liquidations*
+_Liquidations_
 
-[SIP-2005](https://sips.synthetix.io/sips/sip-2005/) introduced endorsed liquidation keepers to perps v2 to ensure that large positions could be liquidated in an MEV-resistant manner (e.g. sandwiching of liquidation transactions). In Perps v3, large positions are gradually liquidated one non-sandwichable portion at a time with a configurable delay in between partial liquidations. This eliminates profit motives to sandwich liquidation transactions, and allows for balancing arbitrages in between partial liquidations. 
+[SIP-2005](https://sips.synthetix.io/sips/sip-2005/) introduced endorsed liquidation keepers to perps v2 to ensure that large positions could be liquidated in an MEV-resistant manner (e.g. sandwiching of liquidation transactions). In Perps v3, large positions are gradually liquidated one non-sandwichable portion at a time with a configurable delay in between partial liquidations. This eliminates profit motives to sandwich liquidation transactions, and allows for balancing arbitrages in between partial liquidations.
 
-*Deterministic settlements*
+_Deterministic settlements_
 
 Delayed orders in Perps v2 have proven extremely effective in minimizing toxic flow, however the details of the implementation can be improved to produce a more resilient settlement flow. In short, the v2 flow requires a price update from after a configurable delay elapses to settle an order. During times of network congestion, this can result in orders being settled with prices from 30-60 seconds post commitment. In perps v3, the settlement process enforces that the execution price is the earliest available valid price (prices with timestamp greater than or equal to `commitment + delay` are considered valid)
```

Comment:
In perps v3, the settlement process enforces that the execution price is the earliest available valid price (prices with timestamp greater than or equal to `commitment + delay` are considered valid)

Fix to erc7412 wrapper orchestration:
In perps v3, the settlement process enforces that the execution price is the price at commitment time and are executed (settled) in the available window (`commitment + delay`, `commitment + delay + window length`)